### PR TITLE
Fixes 4255: Adding container file and renovate.json

### PR DESCRIPTION
**Pull request by mayurilahane**
## Summary

Adding container file and renovate.json file 

As per my understanding : 

The **container file** configures the Docker steps needed to run the frontend of content-sources on Konflux.

And **renovate.json** is to balance the version of tasks it updates the versions on weekly basis.

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-4255

File: .tekton/content-sources-frontend-pull-request.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: tekton.dev/v1
 kind: PipelineRun
 metadata:
   annotations:
-    build.appstudio.openshift.io/repo: https://github.com/content-services/content-sources-frontend?rev={{revision}}
+    build.appstudio.openshift.io/repo: https://github.com/RHEnVision/content-sources-frontend?rev={{revision}}
```
Comment (ezr-ondrej) [inline .tekton/content-sources-frontend-pull-request.yaml:5]: This is probably wrong repo :)

File: .tekton/content-sources-frontend-push.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: tekton.dev/v1
 kind: PipelineRun
 metadata:
   annotations:
-    build.appstudio.openshift.io/repo: https://github.com/content-services/content-sources-frontend?rev={{revision}}
+    build.appstudio.openshift.io/repo: https://github.com/RHEnVision/content-sources-frontend?rev={{revision}}
```
Comment (ezr-ondrej) [inline .tekton/content-sources-frontend-push.yaml:5]: Ditto

**Review by ezr-ondrej — COMMENTED** [review]
Lgtm, apart of wrong repo.

**Pull request closed without merging**